iphone/cli/commands/_build.js

```diff
@@ -2588,6 +2588,11 @@ iOSBuilder.prototype.checkIfNeedToRecompile = function checkIfNeedToRecompile()
 			return true;
 		}
 
+		if (this.target === 'dist-adhoc' || this.target === 'dist-appstore') {
+			this.logger.info(__('Forcing rebuild: distribution builds require \'xcodebuild\' to be run so that resources are copied into the archive'));
```

Comment:
Capitalize `distribution`. We have a lot of logs that look "unformatted", so we should at least avoid it for new ones.